modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js

```diff
@@ -0,0 +1,144 @@
+import {useQuery} from '@apollo/client';
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayModal from '@clayui/modal';
+import React, {useMemo, useState} from 'react';
+import Table from '../../../../common/components/Table';
+import {getAccountSubscriptionsTerms} from '../../../../common/services/liferay/graphql/queries';
+import {status} from '../../utils/constants';
+import StatusTag from '../StatusTag';
+
+const ModalCardSubscription = ({accountSubscriptionERC, observer, onClose}) => {
+	const [activePage, setActivePage] = useState(1);
+	const MAX_ITEMS_PER_PAGE = 5;
+
+	const parseDate = (rawDate) => {
+		const date = new Date(rawDate);
+		const month = date.getMonth() + 1;
+		const day = date.getDate();
+		const year = date.getFullYear();
+
+		return !rawDate
+			? ''
+			: `${day}/${month < 10 ? `0${month}` : month}/${year}`;
```

Comment:
1. The date format would be MM/dd/yyyy?

2. Should be good use toLocaleString eg:
```
const date = new Date('2019-06-01T08:00:00.000');

// "6/1/2019, 8:00:00 AM"
date.toLocaleString('en-US');
// "6/1/2019" with no time portion
date.toLocaleDateString();

// But you can still include `hours` and `minutes` using options
// with `toLocaleDateString()`.
date.toLocaleDateString('en-US', {
  month: 'long',
  day: '2-digit',
  year: 'numeric',
  hour: '2-digit'
}); // "June 01, 2019, 8 AM"
```

Reply:
Fixed, changes are on commit **656f3c5**, _LPS-140164 Implemented date formatter functions_.